shared/desktop/app/menu-helper.desktop.tsx

```diff
@@ -1,16 +1,14 @@
-import * as SafeElectron from '../../util/safe-electron.desktop'
+import * as Electron from 'electron'
 import {isDarwin} from '../../constants/platform'
-import {hideDockIcon} from '../app/dock-icon.desktop'
 
 let devToolsState = false
 
 const windowQuit = () => {
-  SafeElectron.getApp().emit('close-windows')
-  hideDockIcon()
+  Electron.app.emit('KBkeybase', '', {type: 'closeWindows'})
 }
 
-export default function makeMenu(window: any) {
-  const editMenu = {
+export default function makeMenu(window: Electron.BrowserWindow) {
+  const editMenu = new Electron.MenuItem({
```

Comment:
the types that ship with electron only allow partial menu definitions if using the constructor. else you have to pass every value in, and some types are wider than they should be (see `role`), so we just use the constructor version